core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20210407_add_entity_attr_temp_table_procedure.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="mbemis" id="create_entity_attr_temp_table">
+        <createProcedure>
+            CREATE PROCEDURE createEntityAttributeTempTable ()
+            BEGIN
+            create temporary table ENTITY_ATTRIBUTE_TEMP (
```

Comment:
What do you think about adding `ON COMMIT DROP` to this statement? Unsure if that is better or worse than dropping the table in `inTransactionWithAttrTempTable`

Reply:
my $.02: I wasn't sure how `ON COMMIT DROP` would work, since we're creating the temp table outside of/before the calling code's transaction, but need it to persist though that transaction. If testing shows it works as expected, great!